LAKMobile/src/components/AppText 2.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
I think some `src/*` files got duplicated. For example, there are `AppText 2.tsx`, `LabelWrapper 2.tsx`, etc. Was this intended? If not, can we remove them?